package-lock.json

```diff
@@ -1424,6 +1424,62 @@
       "integrity": "sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==",
       "dev": true
     },
+    "@chainlink/contracts-0.0.10": {
+      "version": "npm:@chainlink/contracts@0.0.10",
```

Comment:
mentioned this before quite a number of times, don't commit package.lock if there is no update to any dependances in package.json in current pr

Reply:
ok